Controller/ConfigController.php

```diff
@@ -220,7 +222,7 @@ public function deleteAction(Request $request, $id)
      */
     protected function createNewForm(Config $data, AdapterInterface $adapter)
     {
-        $form = $this->createForm('channel_config_new', $data, [
+        $form = $this->createForm($this->get('integrated_channel.form.config.new.type.proxy'), $data, [
```

Comment:
@jansanne it is indeed not compatible with SF3.0. Since the Proxy Form is going to be removed I have changed this code to "normal SF form definitions".

I will not change the edit / new logic because it has not much to do with the SF2.8 edits. I suggest to create a new issue if you want this to be changed.